app/routes/planets_route.py

```diff
@@ -0,0 +1,96 @@
+from flask import Blueprint, abort, make_response, request, Response
+from ..db import db
+from app.models.planets import Planet
+
+planets_bp = Blueprint("planets_bp", __name__, url_prefix="/planets")
+
+@planets_bp.post("")
+def create_planet():
+    request_body = request.get_json()
+    name = request_body["name"]
+    description = request_body["description"]
+    size = request_body["size"]
+
+    new_planet = Planet(name=name, description=description, size=size)
```

Comment:
If we had a `from_dict` method, we could condense all of this code into a single line!

This is also a great candidate for error handling in a refactoring scenario!